### PR TITLE
Native app styles update

### PR DESCRIPTION
**Pull request by lee-cheung**
I have done the styling for the login page and the header for the taken and remaining holidays. I have added styled-components. Need to add styled components to the rest of the application. Thought I would push this so that we can start on the functionality for the request holiday button and the logout button.

### CODE REVIEW

File: NativeApp/src/components/Login/Form/index.js [hunk 1]
```diff
@@ -2,49 +2,85 @@ import React, { Component } from 'react';
...
+            </Text>
+          )
+        }
+        <TouchableOpacity
+          style={styles.button}
           onPress={() => handleLogin(email, password)}
-          title="Login"
-        />
+        >
+          <Text style={[styles.text]}>
```
Comment (kirsty-unosquare) [inline NativeApp/src/components/Login/Form/index.js]: Think you can remove `[]` around `styles.text`. Only need it if we're passing more than one style object.

File: NativeApp/src/components/Login/View/index.js [hunk 2]
```diff
@@ -1,25 +1,30 @@
...
 
 const LoginView = props => (
-  <ScrollView
-    contentContainerStyle={styles.container}
-  >
-    <View style={styles.card}>
-      <LoginForm {...props} />
-    </View>
-  </ScrollView>
+  <View style={styles.container}>
```
Comment (kirsty-unosquare) [inline NativeApp/src/components/Login/View/index.js]: Are we having just the form scrollable on this screen? I think we should make the entire screen scrollable, cause the form might cut off on smaller screens.

File: NativeApp/src/components/User/View/index.js [hunk 3]
```diff
@@ -3,11 +3,12 @@ import { PropTypes as PT } from 'prop-types';
 import {
   ScrollView,
   View,
-  Text,
+  // Text,
```
Comment (kirsty-unosquare) [inline NativeApp/src/components/User/View/index.js]: Remove?

File: NativeApp/src/components/User/View/index.js [hunk 4]
```diff
@@ -66,13 +91,49 @@ UserView.propTypes = {
...
+    alignItems: 'center',
+
+    borderWidth: 3,
+    borderColor: 'blue',
+  },
+  holidayContainer: {
+    // justifyContent: 'center',
+    // alignSelf: 'center',
+    // flex: 1,
+    // flexDirection: 'row',
```
Comment (kirsty-unosquare) [inline NativeApp/src/components/User/View/index.js]: Remove?

File: NativeApp/src/components/User/View/index.js [hunk 5]
```diff
@@ -66,13 +91,49 @@ UserView.propTypes = {
...
+    borderColor: 'yellow',
+  },
+  holidayText: {
+    flex: 1,
+    flexDirection: 'row',
+    alignItems: 'baseline',
+
+    // justifyContent: 'center',
+    // alignSelf: 'center',
+    borderWidth: 3,
```
Comment (kirsty-unosquare) [inline NativeApp/src/components/User/View/index.js]: Same again.

File: NativeApp/src/utilities/AxiosInstance.js [hunk 6]
```diff
@@ -3,7 +3,8 @@ import Moment from 'moment';
 import deviceStorage from '../services/deviceStorage';
 
 
-const baseURL = process.env.DOMAIN;
+const baseURL = 'http://192.168.1.209:8081'
+//process.env.DOMAIN;
```
Comment (kirsty-unosquare) [inline NativeApp/src/utilities/AxiosInstance.js]: Maybe revert this back into the `process.env.DOMAIN` when pushing up to github. Helps avoids any connection issues. :)

File: NativeApp/theme.js [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+    grey: '#B1B1B1',
+    darkGrey: '#5E6B73',
+  },
+};
+
+// export const Container = styled.container`
+//   flex: 1;
+//   padding-top: 23;
+//   background-color: '#fff';
+// `;
```
Comment (kirsty-unosquare) [inline NativeApp/theme.js]: Remove?

File: NativeApp/package.json [hunk 8]
```diff
@@ -39,6 +39,7 @@
     "react-native-calendars": "^1.19.3",
     "react-native-vector-icons": "^4.6.0",
     "react-navigation": "^2.0.4",
-    "react-navigation-material-bottom-tabs": "^0.3.0"
+    "react-navigation-material-bottom-tabs": "^0.3.0",
```
Comment (kirsty-unosquare) [inline NativeApp/package.json]: I don't think this is being used anymore. Best to remove it if not.

File: NativeApp/.editorconfig [hunk 9]
```diff
@@ -0,0 +1,2 @@
+
+end_of_line = lf
```
Comment (kirsty-unosquare) [inline NativeApp/.editorconfig]: I can't mind, but did this work for us?

**Review by kirsty-unosquare — CHANGES_REQUESTED** [review]
Looks good. Just a couple of changes and some questions. :)

File: NativeApp/src/components/Login/Form/index.js (hunk 1 above)
Comment (lee-cheung) [reply NativeApp/src/components/Login/Form/index.js]: I have changed this to a styled component

File: NativeApp/src/components/Login/View/index.js (hunk 2 above)
Comment (lee-cheung) [reply NativeApp/src/components/Login/View/index.js]: yep, you are right. i have changed this :)

File: NativeApp/src/components/User/View/index.js (hunk 3 above)
Comment (lee-cheung) [reply NativeApp/src/components/User/View/index.js]: done

File: NativeApp/src/components/User/View/index.js (hunk 4 above)
Comment (lee-cheung) [reply NativeApp/src/components/User/View/index.js]: woops thought i removed that. Done :)

File: NativeApp/src/components/User/View/index.js (hunk 5 above)
Comment (lee-cheung) [reply NativeApp/src/components/User/View/index.js]: done

File: NativeApp/src/utilities/AxiosInstance.js (hunk 6 above)
Comment (lee-cheung) [reply NativeApp/src/utilities/AxiosInstance.js]: woops, was meant to do that. done :)

File: NativeApp/theme.js (hunk 7 above)
Comment (lee-cheung) [reply NativeApp/theme.js]: done

File: NativeApp/package.json (hunk 8 above)
Comment (lee-cheung) [reply NativeApp/package.json]: it's now removed :)

File: NativeApp/.editorconfig (hunk 9 above)
Comment (lee-cheung) [reply NativeApp/.editorconfig]: i cant remember. i think it was the .gitattribute file that worked.

Comment (kirsty-unosquare) [reply NativeApp/.editorconfig]: Think your right. I would remove this file then. :)

Comment (lee-cheung) [reply NativeApp/.editorconfig]: cool, removed :)

**Review by kirsty-unosquare — APPROVED** [review]

**Pull request merged**